cmd/root.go

```diff
@@ -45,6 +46,7 @@ func NewRootCmd() *cobra.Command {
 	airflowClient := airflowclient.NewAirflowClient(httputil.NewHTTPClient())
 	astroClient := astro.NewAstroClient(httputil.NewHTTPClient())
 	astroCoreClient := astrocore.NewCoreClient(httputil.NewHTTPClient())
+	platformCoreClient := astroplatformcore.NewCoreClient(httputil.NewHTTPClient())
```

Comment:
again here im wondering if we should call this platformV1beta1CoreClient and call the package astroplatformv1beta1core since we should be expecting newer versions